src/wdlci/auth/refresh_token_auth.py

```diff
@@ -47,7 +62,8 @@ def __obtain_access_token(self):
         response = requests.post(url, params=params, headers=headers)
         if response.status_code != 200:
             raise WdlTestCliExitException(
-                "could not obtain access token from refresh token", 1
+                "The refresh token is not expired, but still could not obtain access token from refresh token. Please ensure the refresh token provided as a GitHub action secret is valid",
```

Comment:
The refresh token could be unexpired upon the initial check, but expired when this is called. I think the original message is probably fine here!